resources/shaders/pp_dof_util.hlsl

```diff
@@ -37,6 +37,7 @@ float GetLinearDepth(float depth)
 {
 	float z = (2 * FNEAR) / (FFAR + FNEAR - (depth * (FFAR - FNEAR)));
 	return z;
+	//return depth * FFAR;
```

Comment:
Either get rid of commented code or add a comment that tells us why you keep that commented code there.